repos/coreclr.proj

```diff
@@ -19,6 +19,7 @@
 
     <CleanCommand>$(ProjectDirectory)/clean$(ShellExtension)</CleanCommand>
     <OfficialBuildId>20181014-03</OfficialBuildId>
+    <PackagesOutput>$(ProjectDirectory)/bin/Product</PackagesOutput>
```

Comment:
Do you know if this output dir is controlled by Arcade? Will need to make Arcade respect repo API or something like it....

Reply:
Generally speaking, Arcade controls these, but as far as I can tell, CoreCLR doesn't use Arcade and still respects the Repo API. I'm going to remove this change.